srml/council/src/voting.rs

```diff
@@ -129,32 +158,36 @@ decl_event!(
 	pub enum Event<T> where <T as system::Trait>::Hash {
 		/// A voting tally has happened for a referendum cancellation vote.
 		/// Last three are yes, no, abstain counts.
-		TallyCancelation(Hash, u32, u32, u32),
+		TallyCancellation(Hash, u32, u32, u32),
 		/// A voting tally has happened for a referendum vote.
 		/// Last three are yes, no, abstain counts.
 		TallyReferendum(Hash, u32, u32, u32),
```

Comment:
as per previous comment, consider changing to `TallyReferendum(Hash, YaysCount, NaysCount, AbstainsCount),`